HarpGuru/HarpFace/HarpFaceFragment/styles/index.ts

```diff
@@ -1,15 +1,30 @@
+//import { useGlobal } from 'reactn'
 import { StyleSheet } from 'react-native'
+import type { HarpStrata } from 'harpstrata'
 
 import type { HarpFaceFragmentProps, HarpFaceFragmentStyles } from '../types'
 import { getFragmentFacts } from '../getFragmentFacts'
 import { columnWidth, rowHeight } from '../../styles'
 import { getHarpFaceFacts } from '../../helpers'
+import type { SetActiveHarpStrata } from '../../../helpers'
 
-export const getStyles = (
-  props: HarpFaceFragmentProps
-): HarpFaceFragmentStyles => {
+type Props = HarpFaceFragmentProps & {
+  readonly activeHarpStrata: HarpStrata
+  readonly setActiveHarpStrata: SetActiveHarpStrata
+}
+export const getStyles = (props: Props): HarpFaceFragmentStyles => {
+  // TODO: understand why the linter seems to think that this is
```

Comment:
Remove this TODO and the associated commented out code. It does seem like `useGlobal` can be used outside of normal hooks limitations. I already knew that `setGlobal` was free from these restrictions. I suppose it might be because there is only one global store, there is no need for the same rendering order requirements which proper hooks have. I wonder if these are in fact not really hooks. I should have a look at the source code.